.packit.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
     trigger: pull_request
```

Comment:
Can you also drop the "Ignore" comment above, please? The trigger was updated in 31ca77cea6df2aa4c2c6cf31aa86c8faa7fa673f .